crates/runtime/src/vmcalls/transfer.rs

```diff
@@ -0,0 +1,34 @@
+use crate::FuncEnv;
+use svm_state::AccountStorage;
+use svm_types::{Address, BytesPrimitive};
+
+/// Sends coins from the current executing account to a destination account.
+///
+/// # Panics
+///
+/// Panics when the destination account does not exist.
+pub fn svm_transfer(env: &FuncEnv, src_addr: i32, dst_addr: i32, amount: i64) {
+    let borrow = env.borrow();
+    let storage = borrow.storage();
+
+    let src_addr = Address::repeat(src_addr as u8);
+    let dst_addr = Address::repeat(dst_addr as u8);
```

Comment:
This is not correct.

Reply:
Can you further elaborate?

Reply:
`Address::repeat` takes a `u8` and creates a new `Address` having 20 bytes looking the same - 
It's useful for testing. 

The `src_addr` and `dst_addr` should have acted as pointers to a Memory cell (pointing to the first byte of an `Address`)